alphalens/utils.py

```diff
@@ -680,8 +640,36 @@ def std_conversion(period_std):
         standard deviation/error values.
     """
     period_len = period_std.name
-    return period_std / np.sqrt(period_len)
+    conversion_factor = (pd.Timedelta(period_len) /
+                         pd.Timedelta(one_period_len))
+    return period_std / np.sqrt(conversion_factor)
 
 
 def get_forward_returns_columns(columns):
-    return columns[columns.astype('str').str.isdigit()]
+    pattern = re.compile(r"^(\d+([Dhms]|ms|us|ns))+$", re.IGNORECASE)
+    valid_columns = [(pattern.match(col) is not None) for col in columns]
+    return columns[valid_columns]
+
+
+def timedelta_to_string(timedelta):
```

Comment:
Does that really not exist inside pandas?

Reply:
Unfortunately no, the pandas version gives back the verbose version. e.g. '1 days 00:00:00'